swarm.go

```diff
@@ -506,6 +506,13 @@ func (s *Swarm) Protocols() (protos []p2p.Protocol) {
 // APIs returns the RPC API descriptors the Swarm implementation offers
 func (s *Swarm) APIs() []rpc.API {
 	apis := []rpc.API{
+		// public APIs
```

Comment:
I'm not sure why we deleted this on the first place.

It seems to be present on the `master` branch, but not the `incentives` branch.

Reply:
I'm not sure what will happen when we have the same `rpc.API` namespace entry twice - `bzz` with public API, and `bzz` with admin API, but since we have it on `master` this probably works fine. I just hope that the second entry (the one below) doesn't override the methods from `&Info{s.config}` - probably worth verifying.

Reply:
I tried methods from both `bzz` APIs and it seems to work OK (`info` vs. `has`, `hive`, etc.).

I don't know what would happen if the two APIs had functions with the same names. So I think it's probably best to change the name of one of them at some point.